index.md

```diff
@@ -57,7 +57,7 @@ Follow the [Getting Started with JanusGraph](http://docs.janusgraph.org/latest/g
 
 JanusGraph is a project under [The Linux
 Foundation](https://www.linux.com/blog/Linux-Foundation-welcomes-JanusGraph),
-and includes participants from Expero, Google, GRAKN.AI, Hortonworks, and IBM.
+and includes participants from Expero, Google, GRAKN.AI, Hortonworks, IBM and Amazon.
```

Comment:
I've been meaning to update this sentence, so good thing you brought this up. We should include other companies and individuals here—e.g., we now have Orion Health and Uber contributing patches as well, as well as unaffiliated individuals.

How about changing this to "IBM, Amazon, and other companies and individuals" to cover everyone?

Also, at some point, this list will also get to be too long, or it will always be incomplete and will be missing some folks who have helped out with the project, so we may remove it altogether, but let's keep it for now.